ui/src/importer/ImporterList.js

```diff
@@ -194,26 +222,32 @@ export class ImporterList extends Component {
     }
   };
 
-  render() {
-    const { isLoading, error, data } = this.state;
+  onPageChange = page => {
+    this.setState({ activePage: page });
+  };
 
-    return isLoading && _isEmpty(error) ? (
+  render() {
+    const { isLoading, error, data, activePage } = this.state;
+    return isLoading && _isEmpty(error) && _isNull(data) ? (
```

Comment:
why not to use isEmpty instead of isNull ? It would be more consistent, and less importing functions
```suggestion
    return isLoading && _isEmpty(error) && _isEmpty(data) ? (
```